### ISSUE TITLE
Add License

### ISSUE THREAD

**Issue by ernesto-jimenez** [darkhelmet/manbearpig#3]
It would be great to have a license for this project.

Most go packages use MIT, so I can do a pull request with a MIT license if you are interested.

**Comment by darkhelmet** [comment]
Good call. I tend to prefer Apache v2, if you wanted to dump that it, that would be great.
